sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -286,7 +286,7 @@ private void mergeDebugImages(final @NotNull SentryEvent event) {
 
     for (String item : proguardUUIDs) {
       DebugImage debugImage = new DebugImage();
-      debugImage.setType("proguard");
+      debugImage.setType(DebugImage.PROGUARD);
```

Comment:
since `MainEventProcessor` now applies the uuid to the event reading from the options, what's about setting the proguardUUID here to the options and let the MainEventProcessor create the DebugImage?
we only do it if the options isn't set though

Reply:
the other thing to consider is, event processor runs for every event, reading the uuid is a 1-time thing, so maybe it'd also make sense to move this logic from here to somewhere else eg `AndroidOptionsInitializer`, the downside is that we do IO in the main thread, maybe we keep this refactor for another PR, we just need to tackle the comment above

Reply:
Here it reads a list of uuids (from all the resources), so it cannot be set on the options, or we change it here to single UUID.

Reply:
we can get just a single id, there's even a comment about that.

https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java#L911-L912